.github/CODEOWNERS

```diff
@@ -36,7 +36,7 @@
 /scripts/upgrade-* @hoverbear
 /scripts/*docker* @MOZGIII
 /scripts/ci-setup-minikube.sh @MOZGIII
-/scripts/ci-setup-helm.sh @MOZGIII
+/scripts/environment/ci-setup-helm.sh @MOZGIII
```

Comment:
Nit: we should probably rename this script to `/scripts/environment/setup-helm.sh`, as it's no longer CI-specific, but rather environment-specific.